tensorbay/utility/attr.py

```diff
@@ -311,6 +314,9 @@ def _get_operators(annotation: Any) -> Tuple[_Callable, _Callable]:
     if isinstance(origin, type) and issubclass(origin, Sequence):
         sequence = origin
         type_ = annotation.__args__[0]
+    elif origin == Union and len(annotation.__args__) == 2 and type(None) in annotation.__args__:
+        sequence = origin
+        type_ = annotation.__args__[0]
```

Comment:
what if the type is `Union[None, xxxx]`, in this case `__args__[0]` will be `NoneType`